sdk/core/System.Net.ClientModel/tests/System.Net.ClientModel.Tests.csproj

```diff
@@ -1,19 +1,28 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   
   <PropertyGroup>
+    <Description>Tests of System.Net.ClientModel types and public contracts.</Description>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <IsTestProject>true</IsTestProject>
+    <Nullable>enable</Nullable>
   </PropertyGroup>
   
   <ItemGroup>
-    <PackageReference Include="xunit.runner.visualstudio" />
-    <PackageReference Include="xunit" />
+    <PackageReference Include="Moq" />
+    <PackageReference Include="NUnit" />
+    <PackageReference Include="NUnit3TestAdapter" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
   </ItemGroup>
   
   <ItemGroup>
     <ProjectReference Include="..\src\System.Net.ClientModel.csproj" />
-    <ProjectReference Include="..\..\Azure.Core\src\Azure.Core.csproj" />
+    <ProjectReference Include="clients\System.Net.ClientModel.Tests.Clients.csproj" />
```

Comment:
Tests of public APIs may need access to test clients in the System.Net.ClientModel.Tests.Clients project.